docs/docs/grants.md

```diff
@@ -1,23 +1,30 @@
-# Ingonyama Grant programs
+# Ingonyama Grants Program 2.0
 
-Ingonyama understands the importance of supporting and fostering a vibrant community of researchers and builders to advance ZK. To encourage progress, we are not only developing in the open but also sharing resources with researchers and builders through various programs.
-
-## ICICLE ZK-GPU Ecosystem Grant
+Exactly one year ago, we launched our first-ever grants program, offering $100,000 to support research involving ICICLE. Today, we’re excited to announce our second grants program — another $100,000 dedicated to researchers. Details below!
```

Comment:
> dedicated to researchers

We should probably rephrase to a larger group than researchers since we say later that students have a category and also anyone adding new primitives to ICICLE or porting current projects